docs/pages/style-spec.js

```diff
@@ -1256,9 +1242,19 @@ export default class extends React.Component {
                                 <a href="#expressions-step"><code>step</code></a> expression, or such an expression within a
                                 <a href="#expressions-let"><code>let</code></a> expression.</p>
 
-                            <h4>Example: a zoom-and-property expression</h4>
-                            <p>Combining zoom and property expressions allows a layer's appearance to change with
-                                both the zoom level <em>and</em> each feature's properties.</p>
+                            <p>There is an important difference between layout and paint properties in
+                                the timing of camera expression evaluation. Paint property camera expressions are
+                                re-evaluated whenever the zoom level changes, even fractionally. For example, a paint
+                                property camera expression will be re-evaluated continuously as the map moves between
+                                zoom levels 4.1 and 4.6. On the other hand, a layout property camera expression is
+                                evaluated only at integer zoom levels. It will <em>not</em> be re-evaluated as the zoom
+                                changes from 4.1 to 4.6 -- only if it goes above 5 or below 4.
+                            </p>
+
+                            <h3>Composite expressions</h3>
```

Comment:
Do we still need a dedicated term for expressions that compose multiple kinds of operators? Or would it suffice for a section titled “Composition” to point out that a single expression can use a mixture of subexpression operators?